appengine/pond/duck/online/js/online.js

```diff
@@ -168,6 +204,14 @@ Pond.Duck.Online.makeRequest = function(url, data, onLoadCallback) {
   xhr.send(data.join('&'));
 };
 
+/**
+ * Submits the form information.
+ * @param {Array<string>} requiredFieldsIds The required ids for the fields we
```

Comment:
Array.<string> (its declared incorrectly in encodeFormElements_ jsdoc too, whoops)